### PR TITLE
OpenAPI Sync

### PR DESCRIPTION
**Pull request by cleb11**

### CODE REVIEW

**Review by chen-annie — APPROVED** [review]

File: openapi/mt_openapi_spec_v1.yaml [hunk 1]
```diff
@@ -11659,22 +11889,6 @@ components:
...
-          description: Value in specified currency's smallest unit. e.g. $10 would
-            be represented as 1000.
-        currency:
-          type: string
-          description: An ISO 4217 conformed currency or a custom currency.
-        currency_exponent:
-          type: integer
-          description: Must be included if currency is a custom currency. The currency_exponent
-            cannot exceed 30.
-          nullable: true
```
Comment (rattrayalex) [inline openapi/mt_openapi_spec_v1.yaml:11677]: @cleb11 was this removal intentional?

Comment (cleb11) [reply openapi/mt_openapi_spec_v1.yaml:11677]: yes
